### PR TITLE
ENT-8220: Added ssh-permit-empty-passwords-no v0.0.1

### PR DESCRIPTION
**Pull request by nickanderson**

### CODE REVIEW

**Comment by nickanderson** [comment]
I believe the failure here is because the dependent module lib-sshd-config is not yet in the index, i expect if the job is re-run after https://github.com/cfengine/build-index/pull/107 is merged that it will green up

**Comment by nickanderson** [comment]
> I believe the failure here is because the dependent module lib-sshd-config is not yet in the index, i expect if the job is re-run after #107 is merged that it will green up

Or if I use the right commit sha it will work :exploding_head: :rage4:

File: cfbs.json [hunk 1]
```diff
@@ -448,6 +448,20 @@
       ]
     },
     "python-lib": { "alias": "library-for-promise-types-in-python" },
+    "sshd-PermitEmptyPasswords-no": {
```
Comment (olehermanse) [inline cfbs.json:451]:
```suggestion
    "ssh-permit-empty-pass-words-no": {
```

I understand that you think this is more technically correct, but I it provides a better user experience if we are consistent, using lowercase names and dashes.

Comment (nickanderson) [reply cfbs.json:451]: I dont like this.

I dont dislike it enough to argue, please make a PR to the repo with the same changes.

Comment (nickanderson) [reply cfbs.json:451]: or, add aliases instead

**Pull request merged**